site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+# Policy Attachment
+
+Many implementations of the API support additional functionality that is not
+covered by the API directly. For example, concepts like timeouts, retries, or
+even CDN configuration are not sufficiently portable to be defined within the
+Gateway API. Instead, implementations may choose to represent these concepts
+with custom Policy resources.
+
+Policies attached to Gateway API resources and implementations must use the
```

Comment:
```suggestion
Policies are attached to Gateway API resources and implementations must use the
```

Is this statement true? Can't one target a policy to a core.Service resource, which is not part of the gw api?

Reply:
Yeah it's hard to describe this accurately, I tried to go with "resources and implementations" to be broad here. This is not meant to limit where these policies may be attached but instead require that policies attached to Gateway API resources and implementations use this pattern. Does that make sense?

Reply:
That makes sense. Being broad makes sense here.